packages/auth-client/src/idleManager.ts

```diff
@@ -114,21 +124,25 @@ class IdleManager {
    * @param {IdleCB} callback function to be called when user goes idle
    */
   public registerCallback(callback: IdleCB): void {
+    if (this._usingDefaultCb) {
+      this.callbacks = [];
```

Comment:
Do I understand correctly that this is currently not working as intended, since `AuthClient.constructor` will always call `registerCallback(logout)`, actually removing the default callback (the page reload) right after creation? Thus IIUC the logout would never be followed by the reload at the moment. 

Side note: Since we need both the logout and the refresh callbacks, it is important that the logout callback is executed before the refresh.

Reply:
Oh, you're right - I'll write a new test for that flow. I'll probably need to update the logic slightly as well